### PR TITLE
Add AWS managed relational sql database support

### PR DESCRIPTION
**Pull request by gareth-ferneyhough**

### CODE REVIEW

**Comment by gareth-ferneyhough** [comment]
Do not merge yet.

File: perfkitbenchmarker/providers/aws/aws_managed_relational_db.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+  def _TeardownNetworking(self):
+    cmd = util.AWS_PREFIX + [
+        'rds',
+        'delete-db-security-group',
+        '--db-security-group-name=%s' % self.security_group_name
+    ]
+    vm_util.IssueCommand(cmd)
+
+  def _Create(self):
+    """Creates the AWS RDS instance"""
```
Comment (stfeng2) [inline perfkitbenchmarker/providers/aws/aws_managed_relational_db.py]: High level question: what's the plan for Aurora instance?

**Comment by stfeng2** [comment]
Just one question regarding Aurora support, rest of code LGTM.

File: perfkitbenchmarker/providers/aws/aws_managed_relational_db.py (hunk 1 above)
Comment (gareth-ferneyhough) [inline perfkitbenchmarker/providers/aws/aws_managed_relational_db.py]: Good question. I plan to add support for Aurora in Q3.

**Comment by stfeng2** [comment]
LGTM, you can merge whenever you are ready.

**Pull request merged**